icons/home-plug.json

```
@@ -0,0 +1,14 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": "jguddas",
```

I don't think we accept single strings here. 🤔
```suggestion
  "contributors": [
    "jguddas"
  ],
```

Original creators of `home` might also need to be listed, and let's not forget the question of renaming `home` => `house`, should we do that, this one too should be `house-plug`.

---

I don't think it should be renamed to house, and, I have not build the new house based on the old house but just did a 100% from scratch approach.

The icon I took the most from as a base is if anything square.

---

The plug is also build from scratch but maybe that's not a good thing.